Sector Ownership/Non-UK/Paris.txt

```diff
@@ -1,26 +1,56 @@
-SECTOR:LFFF Low:0:0
-OWNER:PN:P
+SECTOR:LFFF TP:0:0
+OWNER:PTP:HPKZ:OPKZ:RPAW:P
 
-SECTOR:LFFF Low (DB-295):0:29500
-OWNER:PN:P
-BORDER:LSFF:LFRR_LFFF:LFFF-N:LFFF_EBBU:LDFF01:LDFF02:LDM03:LDM05:LDM07:LDM08:LDM09a:LDM09b:LDM10
+SECTOR:LFFF TP (085-265):8500:26500
+OWNER:PTP:HPKZ:OPKZ:RPAW:P
+BORDER:LSFF:LDM16:LDM11:LDM13:LDM14:LFFF-TP_LFFF-TN:LFFF-TP:LFRR_LFFF-TP
+
+SECTOR:LFFF TN:0:0
+OWNER:PTN:TNTB:AENB:RPAW:P
+
+SECTOR:LFFF TN (085-115):8500:11500
+OWNER:PTN:TNTB:AENB:RPAW:P
+BORDER:LDM01:LDM02:LDFF03:LFFF-TN_LFFF-TB:LFFF-TN:LFFF-TP_LFFF-TN:LDM14:LDM13:LDM11
+
+SECTOR:LFFF TN (115-195):11500:19500
+OWNER:PTN:TNTB:AENB:RPAW:P
+BORDER:LDM12:LDM07:LDM06:LDM03b:LDM03a:LDFF03:LFFF-TN_LFFF-TB:LFFF-TN:LFFF-TP_LFFF-TN:LDM14:LDM13
+
+SECTOR:LFFF TN (195-265):19500:26500
+OWNER:PTN:TNTB:AENB:RPAW:P
+BORDER:LDM08:LDM07:LDM06:LDM03b:LDM03a:LDFF03:LFFF-TN_LFFF-TB:LFFF-TN:LFFF-TP_LFFF-TN
+
+SECTOR:LFFF TB:0:0
+OWNER:PTB:TNTB:AENB:RPAW:P
+
+SECTOR:LFEE TB Delegation (195-265):19500:26500
+OWNER:EEUB:EEN:EE:RPAW:P
+BORDER:LFFF_LFEE_Deleg
```

Comment:
Is it worth a comment to say why this is in Paris rather than Reims? (I assume it's deliberately done like this?)